src/openrct2/config/Config.cpp

```diff
@@ -274,6 +274,7 @@ namespace Config
             model->toolbar_show_research = reader->GetBoolean("toolbar_show_research", true);
             model->toolbar_show_cheats = reader->GetBoolean("toolbar_show_cheats", false);
             model->toolbar_show_news = reader->GetBoolean("toolbar_show_news", false);
+			model->toolbar_show_mute = reader->GetBoolean("toolbar_show_mute", false);
```

Comment:
C++ files use spaces for alignment, rather than tabs (I know, it's confusing because C files do use them, but we're working on that).